src/frontend/src/styles/main.css

```diff
@@ -421,6 +439,7 @@ a:hover,
   height: calc(1em * var(--vs-line-height));
   aspect-ratio: 1;
   vertical-align: middle;
+  transition: color 0.2s ease-out;
```

Comment:
I think transitions on color and opacity should be linear, especially when that short.

Reply:
sure